<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="MainTest(failed)">
    <classes>
      <class name="mavenPractice.tests.mainTestwithTestNG">
        <methods>
          <include name="launchApplication"/>
          <include name="mainTest" invocation-numbers="0 1"/>
          <include name="OrderHistoryTest"/>
        </methods>
      </class> <!-- mavenPractice.tests.mainTestwithTestNG -->
    </classes>
  </test> <!-- MainTest(failed) -->
  <test thread-count="5" parallel="tests" name="ErrorValidationsTest(failed)">
    <classes>
      <class name="mavenPractice.tests.ErrorValidations">
        <methods>
          <include name="productErrorValidation"/>
          <include name="launchApplication"/>
          <include name="loginErrorValidation"/>
        </methods>
      </class> <!-- mavenPractice.tests.ErrorValidations -->
    </classes>
  </test> <!-- ErrorValidationsTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
